Sources/StreamChatUI/ChatChannelList/Tests/ChatChannelListVC_SnapshotTests.swift

```diff
@@ -0,0 +1,66 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+@testable import StreamChatUI
+import XCTest
+
+class ChatChannelListVC_SnapshotTests: XCTestCase {
+    var view: ChatChannelListItemView!
+    var vc: ChatChannelListVC!
+    var mockedChannelListController: ChatChannelListController_Mock<NoExtraData>!
+    
+    override func setUp() {
+        super.setUp()
+        UIConfig.default.channelList.channelListItemSubviews.avatarView = MockAvatarView.self
+        UIConfig.default.currentUser.currentUserViewAvatarView = MockCurrentChatUserAvatarView.self
```

Comment:
In tests we should rather use `UIConfig` provided explicitly. So rather something like this:
```swift
var uiConfig = UIConfig()
uiConfgi.channelList.channelListItemSubviews.avatarView = MockAvatarView.self
uiConfig.currentUser.currentUserViewAvatarView = MockCurrentChatUserAvatarView.self

vc = ChatChannelListVC()
vc.uiConfig = uiConfig
```

Reply:
I tried doing this before, but it is not working properly, the config is not being overridden in that way, maybe we can investigate that in another ticket?

Reply:
I will not use this anyways btw, since a local URL file should fix this, without the need to customize the views.

Reply:
Nuno from the future: The local file did not work, I can see the local URL image exists, but setImage does not render the image properly 🤔 Maybe it tries to use URLSession to download the image under the hood (Nuke)? 

If we keep the MockViews, then can we address this later?
```
vc = ChatChannelListVC()
vc.uiConfig = uiConfig
```
Does not work and does not override the views.

Reply:
The local URL should work. The system doesn't care if the URL is local or remote. -> Slack